config.md

```diff
@@ -289,14 +290,16 @@ For Windows based systems the user structure has the following fields:
 
 ## Platform-specific configuration
 
-[**`platform.os`**](#platform) is used to lookup further platform-specific configuration.
+[**`platform.os`**](#platform) is used to specify platform-specific configuration.
+Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
+Implementations MUST generate a clear error message for valid values it chooses to not support and MUST generate an error when invalid values are encountered.
```

Comment:
> does "a clear error message" mean that the implementation would not continue running and would exit non-zero?

Exiting non-zero on `create` errors is [covered][1] by the in-flight #513.  The currently landed error language has “[as if the operation were never attempted][2]” wording and “[MUST generate an error and a new container MUST NOT be created][3]” wording, which seems to cover everything but the exit-code (which only makes sense in the context of the #513 API).

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-0c1e5a5baa5649945697a530e09acbf5R66
[2]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc4/runtime.md#L69
[3]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc4/runtime.md#L93